public/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css">
+    <script src="https://unpkg.com/ionicons@5.4.0/dist/ionicons.js"></script>
+    <link rel="stylesheet" href="https://unpkg.com/@patternfly/patternfly@4.90.5/patternfly.css" crossorigin="anonymous" />
```

Comment:
```suggestion
    <link rel="stylesheet" href="https://unpkg.com/@patternfly/patternfly@4.90.5/patternfly.css" crossorigin="anonymous" />
        <link rel="stylesheet preload" as="style" href="https://unpkg.com/@patternfly/patternfly/patternfly-addons.css"

```

Try including patternfly addons also it can you you with styling your app and helps to reduce the css weightage of your app

Reply:
I think will add this whenever required and try to utilize in next issue, currently not using any css from it